src/main/java/com/micetr0/mock_DB/MySqlDB.java

```diff
@@ -258,38 +259,106 @@ public Boolean execute(Connection conn) throws SQLException {
 
 
     @Override
-    public void insertNote(Note note) {
+    public void insertNote(String type, String pitch, Integer measureIndex, Integer measureId) {
     }
 
     @Override
-    public void deleteNote(String noteId) {
+    public boolean deleteNote(String noteId) {
+        return false;
     }
 
     @Override
-    public List<Note> findNotesByMeasureIdAndMeasureIndex(Integer measureId, Integer measureIndex) {
+    public List<Composition> findCompositionsByAccountId(Integer accountId) {
         return null;
     }
 
     @Override
-    public List<Composition> findCompositionsIdsByAccountId(Integer accountId) {
+    public List<Note> findNotesByMeasureIdAndMeasureIndex(Integer measureId, Integer measureIndex) {
         return null;
     }
 
     @Override
     public List<Account> findAllAccounts() {
-        return null;
+        return executeTransaction(conn -> {
+            PreparedStatement getAccsStmt = null;
+            ResultSet accResultSet = null;
+
+            try{
+                getAccsStmt = conn.prepareStatement(
+                    "Select accounts.* from accounts"
+                );
+
+                List<Account> resultAccounts = new ArrayList<Account>();
+
+                accResultSet = getAccsStmt.executeQuery();
+
+                //for testing that result was returned i.e. accounts exist
+                Boolean found = false;
+
+                while(accResultSet.next()){
+                    found = true;
+
+                    Account account = new Account();
+                    loadAccount(account, accResultSet, 1);
+
+                    resultAccounts.add(account);
+                }
+
+                if(!found){
+                    System.out.println("No Account was found that matched that username");
+                }
+
+                return resultAccounts;
+            }
```

Comment:
Add a catch block for exceptions.
try -> catch - > finally

Reply:
Catch not needed, nothing to catch